transforms/code/code2parquet/Dockerfile.ray

```diff
@@ -1,41 +1,27 @@
 ARG BASE_IMAGE=docker.io/rayproject/ray:2.24.0-py310
-
 FROM ${BASE_IMAGE}
 
 # see https://docs.openshift.com/container-platform/4.17/openshift_images/create-images.html#use-uid_create-images
 USER root
-RUN chown ray:root /home/ray && chmod g=u /home/ray
+RUN chown ray:root /home/ray && chmod 775 /home/ray
```

Comment:
We encountered permissions issues running on OpenShift and using `g=u` fixed that according to this [link](https://docs.openshift.com/container-platform/4.17/openshift_images/create-images.html#use-uid_create-images). I suggest testing it on Openshift cluster first... Thanks  cc @roytman